FruitFruit/FruitFruit/Controllers/InitSettingViewController.swift

```diff
@@ -140,45 +141,33 @@ class InitSettingViewController: UIViewController {
                     let graident = initSettingButton.applyButtonGradient(colors: Constants.FruitfruitColors.buttonGradient)
                     UIView.animate(withDuration: 4.0, delay: 0, options: .transitionCrossDissolve, animations: {
                         self.initSettingButton.layer.insertSublayer(graident, at: 0)
-                        self.initSettingButton.configuration?.background.backgroundColor = UIColor.clear
-                        self.initSettingButton.titleLabel?.font = UIFont.preferredFont(for: .headline, weight: .bold)
                     }, completion: nil)
                 }
             } else {
                 if initSettingButton.layer.sublayers!.count == 3 {
```

Comment:
```suggestion
                let limitedCount = 3

                if initSettingButton.layer.sublayers!.count == limitedCount {
```

3의 의미가 명명되어있다면 추후에 유지보수에도 도움이 될 듯합니다